schema/rss-service-plan-v5.json

```diff
@@ -18,6 +18,68 @@
     }
   },
   "definitions": {
+    "BlueprintPhysicalDiskConfig": {
```

Comment:
Do we need to bump to RSS service plan v6? This change isn't backwards compatible :grimacing:

Reply:
Damnit, I did consider this, but not closely enough. I was thinking this was one of the `DeployStepVersion`s and that changing the last one wouldn't matter. Practically speaking, I'm not sure this does matter, since the plan only gets reloaded on RSS failing, which only happens on new runs of RSS. But for good measure, I should probably make a v6.

Reply:
I'm actually not sure it makes sense to add another version. The plan is only read during RSS time and we never clean up the chain of reads. This does mean that we'll have 2 formats of v5 in the wild, but the old ones are plans that never get loaded again.

Reply:
@jgallagher and I discussed this, and then we discussed in update huddle. It turns out that the plans aren't really useful at all given that we can't actually restart RSS without a clean-slate. Therefore the current plan (no pun intended) is to get rid of them altogether. I'm going to open an issue about this shortly and then open a PR resolving it.

For now though, I'm not going to go back and add a v6.